.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
`[skip ci]` is now a native [GitHub feature](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/) so I think this is no longer needed.